cartoframes/examples.py

```diff
@@ -0,0 +1,5 @@
+from cartoframes import CartoContext
+
+def exampleContext():
+    """Returns a CartoContext corresponding to a CARTO account with example data"""
+    return CartoContext(base_url='https://cartoframes.carto.com', api_key='default_public')
```

Comment:
Could you limit this line and the docstring above it to 79 lines please?

Reply:
I use both `pep8` and `flake8` for linting purposes

Reply:
Ok! I just enabled pep8 warnings at IntelliJ (I'm trying to use it as swiss-army knife, with language plugins), and [added flask8](https://foxmask.trigger-happy.eu/post/2016/02/17/pycharm-running-flake8/) so it complains:

```
/usr/local/bin/flake8 --max-complexity 10 /Users/juanignaciosl/Development/carto/cartoframes/cartoframes/examples.py
/Users/juanignaciosl/Development/carto/cartoframes/cartoframes/examples.py:5:80: E501 line too long (83 > 79 characters)
/Users/juanignaciosl/Development/carto/cartoframes/cartoframes/examples.py:6:80: E501 line too long (91 > 79 characters)

Process finished with exit code 1
```